content/docs/1.5.1/volumes-and-nodes/internal-data-structures.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Internal Data Structures
```

Comment:
`Disk Data Structures`?

BTW, we should have one for place for engine images, backup and backing image. Create a corresponding tickets to work on later. Jack or Shuo can help with that.

Reply:
I think we need a specific directory for these

Reply:
@shuo-wu

Agree with the idea.
Should we create a new `Chapter` for the docs? What title is better?
 
cc @ejweber @james-munson